specification.markdown

```diff
@@ -869,18 +869,18 @@ An EBML Reader may discard some or all data if the following errors are found in
 
 - Invalid Element Data Size values (e.g. extending the length of the EBML Element beyond the scope of the Parent Element; possibly triggering access-out-of-bounds issues).
 - Very high lengths in order to force out-of-memory situations resulting in a denial of service, access-out-of-bounds issues etc.
-- Missing EBML Elements that are mandatory and have no declared default value.
+- Missing EBML Elements that are mandatory in a Master Element and have no declared default value, making the semantic invalid at that Master Element level.
 - Usage of invalid UTF-8 encoding in EBML Elements of UTF-8 type (e.g. in order to trigger access-out-of-bounds or buffer overflow issues).
-- Usage of invalid data in EBML Elements with a date type.
+- Usage of invalid data in EBML Elements with a date type, trigerring bogus date accesses.
```

Comment:
Are you referring to use of NaN or infinite values in date floating points? We could use more specific in the date definition and say that the date MUST be a finite number and that non-finite numbers as dates are a security issue.

Reply:
Nope, dates are signed integers. But they can refer to dates way back which wouldn't make sense to seek to (for example).